SMLHelper/Handlers/CraftDataHandler_Subnautica.cs

```diff
@@ -235,36 +235,36 @@ ITechData ICraftDataHandler.GetModdedTechData(TechType techType)
         TechData ICraftDataHandler.GetTechData(TechType techType)
         {
 
-            if (CraftDataPatcher.CustomTechData.TryGetValue(techType, out ITechData moddedTechData))
+            if (CraftDataPatcher.CustomTechData.TryGetValue(techType, out ITechData iTechData))
             {
-                TechData techData = new TechData() {craftAmount = moddedTechData.craftAmount};
+                TechData techData = new TechData() {craftAmount = iTechData.craftAmount};
 
-                for(int i = 0; i < moddedTechData.ingredientCount; i++)
+                for(int i = 0; i < iTechData.ingredientCount; i++)
                 {
-                    IIngredient ingredient = moddedTechData.GetIngredient(i);
+                    IIngredient ingredient = iTechData.GetIngredient(i);
                     Ingredient smlingredient = new Ingredient(ingredient.techType, ingredient.amount);
                     techData.Ingredients.Add(smlingredient);
                 }
-                for (int i = 0; i < moddedTechData.linkedItemCount; i++)
+                for (int i = 0; i < iTechData.linkedItemCount; i++)
                 {
-                    techData.LinkedItems.Add(moddedTechData.GetLinkedItem(i));
+                    techData.LinkedItems.Add(iTechData.GetLinkedItem(i));
                 }
                 return techData;
             }
 
-            ITechData td = CraftData.Get(techType, true);
-            if(td != null)
+            iTechData = CraftData.Get(techType, true);
+            if(iTechData != null)
             {
-                TechData techData = new TechData() { craftAmount = td.craftAmount };
-                for (int i = 0; i < td.ingredientCount; i++)
+                TechData techData = new TechData() { craftAmount = iTechData.craftAmount };
+                for (int i = 0; i < iTechData.ingredientCount; i++)
                 {
-                    IIngredient ingredient = td.GetIngredient(i);
+                    IIngredient ingredient = iTechData.GetIngredient(i);
                     Ingredient smlingredient = new Ingredient(ingredient.techType, ingredient.amount);
                     techData.Ingredients.Add(smlingredient);
                 }
-                for (int i = 0; i < td.linkedItemCount; i++)
+                for (int i = 0; i < iTechData.linkedItemCount; i++)
                 {
-                    techData.LinkedItems.Add(moddedTechData.GetLinkedItem(i));
+                    techData.LinkedItems.Add(iTechData.GetLinkedItem(i));
```

Comment:
@MrPurple6411 You had bug there because you copy-pasted code. It is better to just use local function for these two identical pieces of code.

Reply:
Good call. I'm not sure how I didn't see that myself before.

No need to update the release package on Nexus as it will be functionally identical to this refactor.